packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -3,48 +3,42 @@ title: Integrate Interface X Archetype into an existing website
 tags:
   - integration
   - archetype
-  - X Components archetype integration
+  - archetype integration
   - x integration
   - interface x
   - x components
-  - archetype integration
 ---
 
 # Integrate Interface X Archetype into an existing website
 
-Once you have finished developing or extending your search interface using the
-Interface&nbsp;X&nbsp;Archetype project you will probably want to integrate it into your current
-store.
+In this tutorial, you'll learn how to integrate the Interface&nbsp;X&nbsp;Archetype project in your commerce store in a matter of minutes. You can use the X&nbsp;Archetype as is or you can [extend the search and discovery interface experience](web-archetype-development-guide.md) to meet your business needs.
 
-The integration is a 2-steps process:
+To integrate the Interface&nbsp;X&nbsp;Archetype layer in your commerce store, just **load** the generated Interface&nbsp;X JavaScript file and **initialize** it.
 
```

Comment:
We can include here a info Panel with info like this:

If you are loading an archetype script hosted by empathy, X resources are served by a CDN through the following URLs Environment:

- Live: https://x.empathy.co/{INSTANCE}/app.js
- Staging: https://x.staging.empathy.co/{INSTANCE}/app.js

where {INSTANCE} is the name of your store accorded with Empathy.

Reply:
Just curious, has this instance something to do with the instances a partner can configure with the CMC tool from Playboard, @herrardo? See https://docs-dev.empathy.co/explore-empathy-platform/configure-empathy-platform/request-instance.html

Reply:
I think so, you create the instance, or ask to Empathy to create it for you in fact, and then you set here the InstanceId, but i wouldn't distract this developer with things about the playboard, because Playboard is not a thing that this Frontend integrator developer is going to manage IMO

Reply:
Included as a note including a call to support in case of help requesting the instance.